tutorials/plot_metadata_epochs.py

```diff
@@ -0,0 +1,118 @@
+"""
+Pandas querying and metadata with Epochs objects
+------------------------------------------------
+
+Demonstrating pandas-style string querying with Epochs metadata.
+
+Sometimes you've got a more complex trials structure that cannot be easily
+summarized as a set of unique integers. In this case, it may be useful to use
+the ``metadata`` attribute of ``Epochs`` objects. This must be a pandas
+``DataFrame`` where each row is an epoch, and each column corresponds to a
+metadata attribute of each epoch. Columns must be either strings, ints, or
+floats.
+
+First we'll load some data - subjects were presented with individual words
+on a screen, and information about each word (e.g., word frequency) was
+collected in a metadata object.
+
+Loading the data
+================
+First we'll load the data...this is unnecessarily complex right now because
+the data is stored as a dataframe...we should fix this :-)
```

Comment:
we can always use another dataset too - we just need something that's got complex trial structure. I can probably make my data available for this but it is ECoG and we probably want something that's MEG/EEG for a tutorial like this (at least until ECoG is more of a first-class citizen in MNE)